_layouts/people.html

```diff
@@ -54,9 +54,9 @@ <h2>Details</h2>
 <!-- extract experts and organizers -->
 {% assign ols-1-experts = site.data.ols-1-metadata.experts | join: ' ' %}
 {% assign ols-1-organizers = site.data.ols-1-metadata.organizers | join: ' ' %}
-<!-- extract speakers and call hosts -->
+<!-- extract speakers and call facilitators -->
 {% assign ols-1-speakers = '' %}
-{% assign ols-1-hosts = '' %}
+{% assign ols-1-facilitators = '' %}
```

Comment:
Facilitators are in the `ols-x-metadata` file, starting from cohort 3 (no facilitators before)

Reply:
Ah, yes. That's true. Can I go ahead and remove all the lines of code that attempt to extract the facilitators in cohort 1 and 2?

Reply:
Sure

Reply:
Done. Thanks for being so incredibly patient. ❤️